tools/watch_logs.py

```diff
@@ -82,13 +82,14 @@ def print_col(line):
         print_col(line)
 
 
-colors = ['default', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'red']
-threads = [
-    threading.Thread(target=print_log, args=(logfile, colors[n % len(colors)]))
-    for (n, logfile) in enumerate(watched)
-]
-
-for t in threads:
-    t.start()
-for t in threads:
-    t.join()
+if __name__ == "__main__":
+    colors = ['default', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'red']
+    threads = [
+        threading.Thread(target=print_log, args=(logfile, colors[n % len(colors)]))
+        for (n, logfile) in enumerate(watched)
+    ]
+
+    for t in threads:
```

Comment:
No real change to this code other than to make this `watch_logs.py` importable without also kicking off the in-line log tailing.